lib/mixlib/install/generator/bourne/scripts/platform_detection.sh

```diff
@@ -46,8 +46,13 @@ elif test -f "/etc/Eos-release"; then
 elif test -f "/etc/redhat-release"; then
   platform=`sed 's/^\(.\+\) release.*/\1/' /etc/redhat-release | tr '[A-Z]' '[a-z]'`
   platform_version=`sed 's/^.\+ release \([.0-9]\+\).*/\1/' /etc/redhat-release`
+  
+  if test "$platform" = "rocky linux"; then
+  	source /etc/os-release
+ 	os="${ROCKY_SUPPORT_PRODUCT}"
+  	platform_version="${ROCKY_SUPPORT_PRODUCT_VERSION}"
 
```

Comment:
we can update platform to match the entry in [omnitruck](https://omnitruck.chef.io/platforms) by adding `platform="${ID}"`

Reply:
Its already in place. 
https://github.com/chef/mixlib-install/blob/main/lib/mixlib/install/generator/bourne/scripts/platform_detection.sh#L115

Reply:
since that is in an `elif` block, it won't be applied in this block.